lib/math.ts

```diff
@@ -0,0 +1,91 @@
+enum Priority {
+  Low = 1,
+  High = 2
+}
+let DefaultValue = 0
+
+type Operators = typeof operators
+let operators = {
+  '+': Priority.Low,
+  '-': Priority.Low,
+  '*': Priority.High,
+  '/': Priority.High
+}
+
+let operations = {
+  '+': (a: number, b: number) => a + b,
+  '-': (a: number, b: number) => b - a,
+  '*': (a: number, b: number) => a * b,
+  '/': (a: number, b: number) => b / a
+}
+
+function isContinuedDigit(ch: string): boolean {
+  return /^[0-9]*\.?[0-9]*$/.test(ch)
+}
+
+function getPriority(ch: keyof Operators): number {
```

Comment:
Why do we need it?

Reply:
:thinking: Regular expression is unnecessary, here we merely check if the current character is a continuation of a number (It is need to check if the last element is a number or a point). I can create such a check (without additional function)

```ts
let isContinuedNumber = /^[\d.]+$/.test(str.charAt(i - 1)) 
```

`getPriority` - some wrapper, for more readable then `operators[currentOpeator]`